docs/user/README.md

```diff
@@ -51,7 +51,7 @@ To get Kyma CLI for MacOS or Linux, run the following script from the command li
 
 ```sh
 curl -sL "https://raw.githubusercontent.com/kyma-project/cli/refs/heads/main/hack/install_cli_nightly.sh" | sh -
-kyma@v3
+kyma
 ```
```

Comment:
Same here. Please see lines 3-5, 27-28, and 47.

Reply:
This PR is about renaming the usage of the kyma-cli binary. This is still v3 but the name of the binary is changed. So other comments and phases should not be changed